### PR TITLE
update otel-web config

### PR DESCRIPTION
**Pull request by vinay2897**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@splunk/otel-web-dev-root",
+  "name": "@kloudmate/otel-web-dev-root",
```
Comment (amitava82) [inline package.json:2]: Rename the package `otel-js-web`?

File: packages/web/src/index.ts [hunk 2]
```diff
@@ -440,6 +397,13 @@ export const SplunkRum: SplunkOtelWebType = {
       resource: this.resource,
     });
 
+    fetch(`https://cdn.kloudmate.com/${VERSION}/splunk-otel-web.js`, {
+      method: 'HEAD'
+    }).then(resp => {
+      provider.resource.attributes['country'] = resp.headers.get("Cloudfront-Viewer-Country-Name") || undefined
+      provider.resource.attributes['city'] = resp.headers.get("CloudFront-Viewer-City") || undefined
```
Comment (amitava82) [inline packages/web/src/index.ts:404]: Are we adding client IP as well?

**Review by amitava82 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vinay2897) [reply package.json:2]: ok. But we won't be deploying this package. The one we will publish is a sub-package [here](https://github.com/kloudmate/otel-js-web/blob/update-config/packages/web/package.json#L2)

Comment (amitava82) [reply package.json:2]: okay

File: packages/web/src/index.ts (hunk 2 above)
Comment (vinay2897) [reply packages/web/src/index.ts:404]: it is possible with `CloudFront-Viewer-Address` should I add it?

Comment (amitava82) [reply packages/web/src/index.ts:404]: Yes

File: packages/web/src/index.ts [hunk 3]
```diff
@@ -402,6 +402,9 @@ export const SplunkRum: SplunkOtelWebType = {
     }).then(resp => {
       provider.resource.attributes['country'] = resp.headers.get("Cloudfront-Viewer-Country-Name") || undefined
       provider.resource.attributes['city'] = resp.headers.get("CloudFront-Viewer-City") || undefined
+      const clientAddress = resp.headers.get("CloudFront-Viewer-Address")?.split(':')
+      provider.resource.attributes['client.ip'] = clientAddress?.[0]
+      provider.resource.attributes['client.port'] = clientAddress?.[1]
```
Comment (amitava82) [inline packages/web/src/index.ts:407]: I don't think we need to split up. Just save client address as is. May be just whole viewer address as is? Sometimes if there is proxy it'll have multiple address

Comment (vinay2897) [reply packages/web/src/index.ts:407]: From the [doc](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/adding-cloudfront-headers.html#cloudfront-headers-viewer-location) it seems it is a single address.

**Review by amitava82 — APPROVED** [review]

**Pull request merged**